### PR TITLE
Update layers version to 1.0.0-alpha2 and use tf.batchNorm instead of tf.batchNormalization.

### PR DESCRIPTION
**Pull request by nsthorat**
This change is [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs-layers/453)

### CODE REVIEW

**Comment by caisq** [comment]
Can you check the test failures?

**Comment by nsthorat** [comment]
Forgot to run make-version, yay unit tests
